src/components/team/member-table.tsx

```diff
@@ -1,59 +1,98 @@
+      <div className="bg-white shadow overflow-hidden sm:rounded-md mt-2">
+        <ul className="divide-y divide-gray-200 dark:divide-gray-700">
+          {members.map((member: any, i: number) => {
+            const {id, name, email, roles, date_added} = member
+            const isOwner = roles.includes('Owner')
 
-          return (
-            <li
-              key={id}
-              className={
-                i % 2 === 0
-                  ? 'bg-gray-100 dark:bg-gray-800'
-                  : 'bg-white dark:bg-gray-900'
-              }
-            >
-              <div className="px-4 py-4 sm:px-6">
-                <div className="flex items-center justify-between">
-                  <p className="text-sm font-medium truncate">
-                    {name ? (
-                      <span>
-                        {name} ({email})
-                      </span>
-                    ) : (
-                      <span>{email}</span>
-                    )}
-                  </p>
-                  <div className="ml-2 flex-shrink-0 flex">
-                    {/* <p className="cursor-pointer px-2 inline-flex text-xs leading-5 font-semibold rounded-full bg-red-100 dark:bg-red-700 text-red-800 dark:text-red-100 hover:shadow"> */}
-                    {/*   Remove */}
-                    {/* </p> */}
-                  </div>
-                </div>
-                <div className="mt-2 sm:flex sm:justify-between">
-                  <div className="sm:flex">
-                    <p className="flex items-center text-sm text-gray-500 dark:text-gray-300">
-                      {roles.join(', ')}
+            return (
+              <li
+                key={id}
+                className={
+                  i % 2 === 0
+                    ? 'bg-gray-100 dark:bg-gray-800'
+                    : 'bg-white dark:bg-gray-900'
+                }
+              >
+                <div className="px-4 py-4 sm:px-6">
+                  <div className="flex items-center justify-between">
+                    <p className="text-sm font-medium truncate">
+                      {name ? (
+                        <span>
+                          {name} ({email})
+                        </span>
+                      ) : (
+                        <span>{email}</span>
+                      )}
                     </p>
+                    <div className="ml-2 flex-shrink-0 flex">
+                      {!isOwner && (
+                        <p
+                          onClick={() => {
+                            send({type: 'OPEN_DIALOG', payload: {member}})
+                            // removeTeamMember(id)
+                          }}
+                          className="cursor-pointer px-2 inline-flex text-xs leading-5 font-semibold rounded-full bg-red-100 dark:bg-red-700 text-red-800 dark:text-red-100 hover:shadow"
+                        >
+                          Remove
+                        </p>
+                      )}
```

Comment:
Remove button, triggers the Confirm Dialog to open. This 'remove' button does not show up for owners, only members can be removed.